eras/conway/impl/testlib/Test/Cardano/Ledger/Conway/ImpTest.hs

```diff
@@ -350,7 +351,7 @@ submitFailingVote ::
   ) =>
   Voter (EraCrypto era) ->
   GovActionId (EraCrypto era) ->
-  [PredicateFailure (EraRule "LEDGER" era)] ->
+  NonEmpty (PredicateFailure (EraRule "LEDGER" era)) ->
```

Comment:
This package, despite introducing a breaking change will not need a version bump, since it has already been increased. Just FYI.

Reply:
Changelog updated